examples/tests/error/utilities.w

```diff
@@ -0,0 +1,4 @@
+assert(false);
+print("W");
```

Comment:
We need some tests that assert some expected results. For example, where can I see  the output of each one of these?

I understand from @MarkMcCulloh that there's a plan to snapshot errors so maybe that's going to be covered there. But this is something we should prioritize because those are not really sufficient to verify the behavior of these functions in my view.

Reply:
I would love to have typed error checking in whatever solution we end up with. For example if code is expected to throw the following Error:
```ts
class WingCliException extends Error {}
```
It needs to have an assertion like this:
```ts
expect(call()).rejects.toThrowWith(WingCliException);
```